src/index.js

```diff
@@ -0,0 +1,52 @@
+import './style.css';
+import createData from './modules/createData.js';
+import readDataFromDb from './modules/readData.js';
+import saveToLocalStorage from './modules/saveToDb.js';
+import renderData from './modules/render.js';
+import editData from './modules/editData.js';
+
+class TodoApp {
+  constructor() {
+    this.todoCollection = readDataFromDb();
+    this.userInput = document.getElementById('todoInput');
+    this.todoContainer = document.querySelector('.list-container');
+    this.userInput.addEventListener('keypress', (event) => {
+      if (event.key === 'Enter') {
+        event.preventDefault();
+        this.CreateData();
+        saveToLocalStorage(this.todoCollection);
+        this.ShowData();
+      }
+    });
+  }
+
+  CreateData() {
+    createData(this.todoCollection);
```

Comment:
While creating the tasks, the index of the last task should correspond with the length of the array. Hence, the index should start from 1 instead of 0. Kindly fix this issue. I have recorded [this video](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md) where I demonstrated what exactly is wrong.